restli-server/src/main/java/com/linkedin/restli/server/config/ResourceMethodConfig.java

```diff
@@ -21,4 +21,9 @@
    * Config for whether this method will need to validate query parameters.
    */
   boolean shouldValidateQueryParams();
+
+  /**
+   * Config for whether this method will need to validate path/resource keys.
+   */
+  boolean shouldValidateResourceKeyParams();
```

Comment:
Replying to your comment on the last PR:

> Resource keys are identifiers for entity on which the resource takes action on.
> there are two configs here one to validate keys in the path, 2nd in method level config which validates keys in batch requests or resource method argument/params

So are you saying that the service-level config only determines whether to validate path keys? And the method-level config determines whether to validate the key (either from the path or query param) passed into the resource method invocation? If so, then this naming is confusing and doesn't seem to align with this logic. If my assessment is correct, then a name like `RestLiConfig#shouldValidatePathKeys` and `RestLiMethodConfig#shouldValidateResourceKeys` are more intuitive.

Also, since these are interfaces meant for public consumption, this should be thoroughly clarified via javadocs on the getters and setters of both configs.

Reply:
shouldValidateResourceKeys at all places. updated docs